app/scripts/directives/addMobileClient.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+(function() {
+  angular.module('openshiftConsole').component('addMobileClient', {
+    controller: [
+      '$scope',
+      'APIService',
+      'DataService',
+      'NotificationsService',
+      AddMobileClient
+    ],
+    controllerAs: 'ctrl',
+    bindings: {
+      onClose: '<',
+      project: '<',
+      serviceName: '<'
+    },
+    templateUrl: 'views/directives/add-mobile-client.html'
+  });
+
+  function AddMobileClient($scope,
+                       APIService,
+                       DataService,
+                       NotificationsService) {
+    var ctrl = this;
+    var mobileclientVersion = {
```

Comment:
wondering should we add mobileService that defines this rather than defining it over and over

Reply:
adding this now